docs/builds/guides/frameworks/angular.md

```diff
@@ -0,0 +1,298 @@
+---
+category: builds-integration-frameworks
+order: 20
+---
+
+# Angular 2+ component
+
+[![npm version](https://badge.fury.io/js/%40ckeditor%2Fckeditor5-angular.svg)](https://www.npmjs.com/package/@ckeditor/ckeditor5-angular)
+
+CKEditor 5 consists of a {@link builds/guides/overview ready to use builds} and a {@link framework/guides/overview CKEditor 5 Framework} upon which the builds are based.
+
+Currently, the CKEditor 5 component for Angular supports integrating CKEditor 5 only via builds. Integrating {@link builds/guides/integration/advanced-setup#scenario-2-building-from-source CKEditor 5 from source} is not yet possible due to the lack of ability to [adjust webpack configuration in `angular-cli`](https://github.com/angular/angular-cli/issues/10618).
+
+<info-box>
+	While there is no support to integrate CKEditor 5 from source yet, you can still {@link builds/guides/development/custom-builds create a custom build of CKEditor 5} and include it in your Angular application.
+</info-box>
+
+## Quick start
+
+In your existing Angular project, install the [CKEditor component](https://www.npmjs.com/package/@ckeditor/ckeditor5-angular):
+
+```bash
+npm install --save-dev @ckeditor/ckeditor5-angular
+```
+
+Install one of the {@link builds/guides/overview#available-builds official editor builds} or {@link builds/guides/development/custom-builds create a custom one} (e.g. if you want to install more plugins or customize any other thing which cannot be controlled via {@link builds/guides/integration/configuration editor configuration}.
+
+Let's say you picked [`@ckeditor/ckeditor5-build-classic`](https://www.npmjs.com/package/@ckeditor/ckeditor5-build-classic):
+
+```bash
+npm install --save-dev @ckeditor/ckeditor5-build-classic
+```
+
+**Note:** You may need to allow external JS in your project's `tsconfig.json` for the builds to work properly:
+
+```json
+"compilerOptions": {
+	"allowJs": true
+}
+```
+
+Now, include the CKEditor module (**TODO WHERE?**):
```

Comment:
cc @ma2ciek

Reply:
Maybe smth like this: `Now, add `CKEditorModule` to your application module imports.`